src/base/random.hh

```diff
@@ -58,22 +58,47 @@ namespace gem5
 
 class Checkpoint;
 
-class Random : public Serializable
```

Comment:
I believe making Random not serializable makes sense but I would add a separate commit for this

Reply:
I will create a PR doing that to enable this PR then.